src/android_components.py

```diff
@@ -150,7 +150,10 @@ def _update_geckoview(
             f"{ts()} Current GV {gv_channel.capitalize()} version in A-C {ac_repo.full_name}:{release_branch_name} is {current_gv_version}"
         )
 
-        current_gv_major_version = major_gv_version_from_version(current_gv_version)
+        if ac_major_version == "main":
+            current_gv_major_version = None
```

Comment:
Nit: Let add a small comment to explain why we're giving this special value. For instance:

```suggestion
            # We always want to be on the latest geckoview version on the main branch.
            current_gv_major_version = None
```

Reply:
Fair.  Added.